esp_jpeg/test_apps/main/test_usb_camera_2_jpg.h

```diff
@@ -0,0 +1,8 @@
+/*
+Raw data from Logitech C170 USB camera was reconstructed to usb_camera_2.jpg
+The it was converted to RGB888 array with jpg_to_rgb888_hex.py
+*/
+
+// JPEG encoded frame 160x120, 1384 bytes, has broken 0xFFFF marker
+extern const unsigned char camera_2_jpg[] asm("_binary_usb_camera_2_jpg_start");
+const unsigned int camera_2_jpg_len = 1384;
```

Comment:
Question, why not calculate the difference between _binary_usb_camera_2_jpg_end and _binary_usb_camera_2_jpg_start? Seems no need to hardcode the number that way.

Reply:
good point, will fix in all embedded files